dev-docs/bidders/adhash.md

```diff
@@ -4,6 +4,7 @@ title: AdHash
 description: Prebid AdHash Bidder Adapter
 pbjs: true
 biddercode: adhash
+media_types: display, video
```

Comment:
I would also recommend to add the `multiformat_supported` (will-bid-on-any, will-bid-on-one, will-not-bid ) meta information.

This indicates how your adapter behaves in the face of multi format ad units (display & video in your case)

Reply:
Thank you. Will check how the bid adapter behaves in that case with our developers and will update the documentation soon.

Reply:
I've added the requested changes.